forms-web-app/tests/unit/validators/eligibility/decision-date.test.js

```diff
@@ -109,17 +110,22 @@ describe('routes/validators/eligibility/decision-date', () => {
       );
     });
 
-    xit('should throw if decision date is too far in the past', () => {
+    it('should throw if decision date is too far in the past', () => {
+      MockDate.set('2021-01-01');
```

Comment:
Probably worth adding `MockDate.reset()` at the end of the test so it doesn't break any other tests

You can achieve this without an addition dependency if you wanted https://dev.to/maxpou/how-to-mock-date-with-jest-3k4b